internal/gensnippets/cmd/gensnippets/main.go

```diff
@@ -30,6 +30,8 @@ import (
 func main() {
 	outDir := flag.String("out", "internal/generated/snippets", "Output directory (default internal/generated/snippets)")
 	googleapisDir := flag.String("googleapis-dir", "", "Root directory of googleapis/googleapis")
+	var scope bool
```

Comment:
I don't think this is used anymore, please remove.

Reply:
Done